### PR TITLE
Click more labels by checkbox

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/po/Job.java [hunk 1]
```diff
@@ -369,7 +371,7 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        control("/properties/hudson-model-ParametersDefinitionProperty/specified").check();
+        control(by.checkbox("This project is parameterized")).check();
```
Comment (timja) [inline src/main/java/org/jenkinsci/test/acceptance/po/Job.java:374]: need a checkbox instance here so we don't unclick this, other places I took a shortcut so didn't need to add the text. (of course other ways to do this)

**Comment by uhafner** [comment]
This PR seems to break my ATH tests:
- https://github.com/jenkinsci/warnings-ng-plugin/pull/1175
- https://github.com/jenkinsci/warnings-ng-plugin/pull/1176
- https://github.com/jenkinsci/warnings-ng-plugin/pull/1168

Any idea what changed so that those tests fail? The title of the PR seems to be unrelated to my tests not finding :
```
Unable to locate By.cssSelector: [path='/hetero-list-add[publisher]'] in http://127.0.0.1:51255/job/notorious_coincidence/configure
```

**Comment by timja** [comment]
Yeah labels are being clicked for checkboxes and radio's now.

Locator's need to use the text rather than direct paths or resolve the label after using the locator.

Core is changing to use hidden inputs and checkboxes to get prettier ones.

**Comment by uhafner** [comment]
But the update to latest core 2.330 still works:
https://github.com/jenkinsci/warnings-ng-plugin/actions/runs/1707048863

**Comment by timja** [comment]
> But the update to latest core 2.330 still works: [jenkinsci/warnings-ng-plugin/actions/runs/1707048863](https://github.com/jenkinsci/warnings-ng-plugin/actions/runs/1707048863)

Yes it's the modernise forms PR which has been pretty much ready to go for nearly a month...

**Pull request merged**